src/examples/block_stream.rs

```diff
@@ -0,0 +1,41 @@
+mod config;
+use std::num::NonZeroU64;
+use iroha_client::client::Client;
+
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let config = config::get_config();
+    let iroha_client: Client = Client::new(&config)?;
+
+    initiate_block_listener(&iroha_client, 1);
+
+    Ok(())
+}
+// Auxiliary method for a block listener
+// You shall implement it first
```

Comment:
```suggestion
}

/// Auxiliary method for a block listener
/// You shall implement it first
```

В Rust мы пишем док-комменты с использованием трех косых черт, чтобы отделить их от обычных комментариев